ReleaseNotes/eng/Altibase_7_3_0_0_1_Release_Notes.md

```diff
@@ -0,0 +1,859 @@
+
+- INDEX_INITTRANS maximum value has been increased from 30 to 50.
+- Default and maximum values for INDEX_MAXTRANS have been changed from 30 to 50.
+
+#### 2.1.14 DBeaver Package
+
+Altibase 7.3 provides a package for DBeaver on Windows.
+
+</br>
+
+### 2.2 Changes
+
+The following describes the features that DBAs and developers need to be aware of, which include additions, modifications, and removals.
+
+#### 2.2.1 Database Version
+
+Version by Database Component
+
+| Altibase Server/Client Version | Database Binary Version | Meta Version | Communication Protocol Version | Replication Protocol Version |
+| :----------------------------: | :---------------------: | :----------: | :----------------------------: | :--------------------------: |
+|           7.1.0.8.8            |          6.5.1          |    8.11.1    |             7.1.7              |            7.4.7             |
+|           7.3.0.0.1            |          7.3.0          |    9.3.1     |             7.1.8              |            7.4.9             |
+
+#### 2.2.2 Compatibility
+
+##### Database Binary Version
+
+The database binary version indicates the compatibility between database image files and log files.
+
+The updated database binary version of Altibase 7.3 is not compatible with the lower version anymore because the logging structure of log files has changed. Therefore, **if users want to upgrade the existing Altibase version to Altibase 7.3, migration tasks are required.**
+
+##### Meta Version
+
+Given that the major version of Meta has changed, **it is necessary to reconfigure the metadata when upgrading from the earlier version to Altibase 7.3**.
+
+##### Communication Protocol Version
+
+The patch version of the communication protocol version has been changed. Altibase 7.3 ensures client backward compatibility only if the versions of major and minor are identical.
+
+> Client backward compatibility ensures that user applications developed using an older version of the Altibase library work well when connected to a newer version of Altibase.
+
+##### Replication Protocol Version
+
+The replication protocol version indicates the compatibility of replication for both backward compatibility and compatibility with additional replication features.
+
+Altibase replication in LAZY mode ensures the Altibase replication backward compatibility because there is no change in major and minor versions of the replication protocol. However, it does not ensure compatibility with replication additional features including EAGER mode replication, because the patch version has been changed.
+
+> ###### Altibase Replication Backward Compatibility
+>
+> Only LAZY mode replication guarantees backward compatibility.
+>
+> While EAGER mode replication DDL synchronization and Offline Replication do not ensure backward compatibility since they require the same replication protocol version.
```

Comment:
```suggestion
> While EAGER mode replication, DDL synchronization and Offline Replication do not ensure backward compatibility since they require the same replication protocol version.
```